local_settings.py.example

```diff
@@ -174,6 +174,32 @@ RELATE_DOCKER_TLS_CONFIG = None
 #     ca_cert=os.path.join(pki_base_dir, "ca.pem"),
 #     verify=True)
 
+# The following lines are for docker-machine on windows
+import platform
+if not platform.system().lower().startswith("linux"):
```

Comment:
I'd rather have this function sitting somewhere in a module within RELATE, to be imported into the config. That way, we can make changes without every user having to update their config. I'd also prefer a naming like `get_docker_host_ip_from_docker_machine` or some such.